homework/java/anzhela_khamitskaya/lecture2/tasks.md

```diff
@@ -0,0 +1,62 @@
+## Описание:
+умение дать название переменной, которая будет хранить в себе значение - это целое искусство. 
+Название должно быть понятным и доступным для другого человека, давать контекстуальную информацию и отражать ее цель. 
+ 
+## Требования: 
+придумать названия для следующих переменных
+
+## 
+Количество кораблей в порту - quantityOfShips
+Счетчик целых чисел - intCounter
+Первое число - firstNumber
+Ответ на вопрос - answer
+Случайное число - randomNumber
+Генератор случайных чисел - generatorOfRandomNumbers
+Цвет автомобиля - carColor
+Скорость движения автомобиля - carSpeed
+Введеное число пользователем - inputOfUser
+Наибольшее число - numberMax
+Наименьшее число - numberMin
+Левая граница диапазона чисел - limitMin
+Правая граница диапазона чисел - limitMax
+Загаданное пользователем число - choiceOfUser
+Вариант ответа - answerOption
+Банковский счет - bankAccount
+Банковский баланс - accontantBalance
+Кредитная карта - credirCardId
+Количество колес у велосипеда - bicycleQuantityOfWheels
+Стоимость билета - ticketPrice
+
+
+## Описание:
+oпределение типа переменной является важной задачей, поскольку тип должен отражать ту сущность, которую нужно хранить,
+а также обозначает набор операций над переменной. Например, при работе с математическими операциями лучше выбирать 
+численный тип данных вместо строк, даже если в строке содержатся числа, поскольку над строкой математические операции
+выполнить нельзя.
+ 
+## Требования:
+укажите тип для следующих переменных (можно указать все подходящие варианты)
+
+##
+Количество кораблей в порту - bite
```

Comment:
Why nof int?

Reply:
> Why nof int?
I offer 127 ships for the port - too much. So I've choice byte which type takes less memory (and have made a grammatical mistake). But most likely you are right: in real projects use int.

Reply:
> > Why nof int?
> > I offer more than 127 ships for the port - too much. So I've choice byte which type takes less memory (and have made a grammatical mistake). But most likely you are right: in real projects use int.